### ISSUE TITLE
Upgrade args Package

### ISSUE THREAD

**Issue by maerlynflagg** [KingWu/gen_lang#24]
after upgrading to Flutter 2.0 and update some package to be compatible with Flutter 2.0, i got this issue:

```
Because flutter_launcher_icons >=0.9.0 depends on args 2.0.0 and gen_lang 0.1.3 depends on args >=0.12.1 <2.0.0, flutter_launcher_icons >=0.9.0 is incompatible with gen_lang 0.1.3.

And because no versions of gen_lang match >0.1.3 <0.2.0, flutter_launcher_icons >=0.9.0 is incompatible with gen_lang ^0.1.3.
```

can you upgrade the args package?

**Comment by maerlynflagg** [comment]
#25

**Comment by maerlynflagg** [comment]
closed, cause #26 Package Update and Null Safety
